x/consensus/types/message_add_evidence.go

```diff
@@ -9,23 +9,10 @@ const TypeMsgAddEvidence = "add_evidence"
 
 var _ sdk.Msg = &MsgAddEvidence{}
 
-func (msg *MsgAddEvidence) Route() string {
-	return RouterKey
-}
-
-func (msg *MsgAddEvidence) Type() string {
-	return TypeMsgAddEvidence
-}
```

Comment:
We're using `Type()` in production code for some of these messages. I don't see that usage removed or changed in this PR. Why remove it in the first place? What's the new approach here?

Reply:
For example here, https://github.com/palomachain/paloma/blob/2226e95be60c4972dff44afbbbe7b4cb0ec268c8/x/gravity/keeper/msg_server.go#L53 - but there's more places in the code.

Reply:
Updated